excercises/02. Function Exercises/01. Calculating Average/Calculate_average_Morme_02.py

```diff
@@ -0,0 +1,16 @@
+def getNumber1():
+    return int(input("Podaj pierwszą liczbę: "))
+
+def getNumber2():
```

Comment:
Dobra, to jeszcze raz - spróbuj wyciągnąć z tych trzech funkcji - bo są bardzo podobne - to, co je różni. Wyciągnij to jako parametr, a te trzy podobne funkcje zwiń do jednej i tylko zmieniaj pierwszą, drugą i trzecią.

Reply:
Ok, zrobiliśmy jeszcze coś innego...